client/src/components/login/ProfilePage.jsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { useAuth0 } from "@auth0/auth0-react";
+import axios from "axios";
+
+const ProfilePage = () => {
+  const { user, isAuthenticated } = useAuth0();
+
+    const storeUserInfo = async () => {
```

Comment:
Thought: maybe we don't need to store this user data at all. Life maybe a little easier if we don't have to use the users table at all we can sync on this tomorrow

Reply:
On Wednesday's stand-up, let's update user info and discuss how to shape the future.